client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/styles.scss

```diff
@@ -0,0 +1,127 @@
+@import "@automattic/onboarding/styles/mixins";
+
+$sensei-color: #43af99;
```

Comment:
Since this variable gets defined multiple times in multiple places with the same value, did you consider creating a new shared file that you would import?

Reply:
Moved to a common scss file in 2b105cd55e551c7b35b817c6e06caa967c148645